docs/portal/src/common/Card/Cardtsx.md

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: 'Card'
+sidebar_position: 1
+---
+# Card
+
+* js code for a single Card component that shows up at [https://dev.tamudatathon.com/events](https://dev.tamudatathon.com/events)
+* uses the Reminder component in src/common/Card/Reminder display time until the activity will be active
```

Comment:
we also show other statuses for the event, ongoing and completed (or something similar)